L1Trigger/DemonstratorTools/plugins/GTTFileWriter.cc

```diff
@@ -225,6 +231,10 @@ void GTTFileWriter::analyze(const edm::Event& iEvent, const edm::EventSetup& iSe
   sumsData.insert(sumsData.end(), htMissDispData.at(0).begin(), htMissDispData.at(0).end());
   sumsData.insert(sumsData.end(), etMissData.at(0).begin(), etMissData.at(0).end());
 
+  std::vector<ap_uint<64>> mesonsData;
+  mesonsData.insert(mesonsData.end(), tripletsData.at(0).begin(), tripletsData.at(0).end());
+  mesonsData.insert(mesonsData.end(), 36, 0);
```

Comment:
I would prefer that we have a `tripletsData` object and insert that rather than using a `mesonsData` object.